swarm/network/stream/delivery.go

```diff
@@ -259,6 +259,11 @@ func (d *Delivery) RequestFromPeers(ctx context.Context, req *network.Request) (
 				//log.Warn("Delivery.RequestFromPeers: peer not found", "id", id)
 				return true
 			}
+			// nodes that do not provide stream protocol
```

Comment:
can this happen at all? i thought `getPeer` only returns stream peers

Reply:
>  make sure this line https://github.com/ethereum/go-ethereum/blob/master/swarm/network/stream/delivery.go#L258 takes care of the bootnode not being requested

Yes, that is correct, can you please add more context to this earlier request? Handling this case is actually not needed.